### PR TITLE
Switching to using Github container registry for base image

### PR DESCRIPTION
**Pull request by hsezhiyan**

### CODE REVIEW

**Review by cloudw — COMMENTED** [review]
Can you include the commands or a link for instructions, for building and pushing docker images here?

**Comment by hsezhiyan** [comment]
> Can you include the commands or a link for instructions, for building and pushing docker images here?
> 
> Edit: I'm referring to comment session at top of file in `metaflow/plugins/kfp/Dockerfile`.

Good comment Yun, just updated with instructions.

**Review by cloudw — APPROVED** [review]

**Review by talebzeghmi — APPROVED** [review]
thank you for documenting!

File: metaflow/plugins/kfp/kfp_constants.py [hunk 1]
```diff
@@ -3,7 +3,9 @@
 # Defaults for running MF on KFP
 import os
 
-BASE_IMAGE = "hsezhiyan/metaflow-zillow:2.1"
+# Link to package:
+# https://github.com/zillow/metaflow/pkgs/container/metaflow%2Fmetaflow-zillow
+BASE_IMAGE = "ghcr.io/zillow/metaflow/metaflow-zillow:2.1"
```
Comment (alexlatchford) [inline metaflow/plugins/kfp/kfp_constants.py:8]: How are we building this image?

Comment (cloudw) [reply metaflow/plugins/kfp/kfp_constants.py:8]: It's probably built manually for now.

Comment (alexlatchford) [reply metaflow/plugins/kfp/kfp_constants.py:8]: Hmm how is it pushed to that Docker repo? Is that manually from @hsezhiyan's machine then too? Might be worthwhile trying to document that process quickly too if possible.

Comment (hsezhiyan) [reply metaflow/plugins/kfp/kfp_constants.py:8]: Hey @alexlatchford , I've documented that [here](https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/plugins/kfp/Dockerfile#L7), let me know if you want anything else.

Comment (alexlatchford) [reply metaflow/plugins/kfp/kfp_constants.py:8]: 🤦  makes sense, thanks!

**Pull request merged**